### DISCUSSION TITLE
Configuration setting to add space between filename and underline at terminal prompt

### DISCUSSION THREAD

**Discussion by geball** [kovidgoyal/kitty#4851] [Q&A] [answered]
I've switched from Alacritty to kitty and the only configuration I can't locate is the padding between a filename and the underline. Alacritty puts the underline below the text descenders while kitty seems to put it tight against the baseline.

This is Alacritty:
<img width="318" alt="Filename underlining in Alacritty with padding that drops it below the text descenders." src="https://user-images.githubusercontent.com/45116528/159133956-49e6e3d1-5035-4b50-a442-3c3389c1e35f.png">

This is kitty:
<img width="300" alt="Filename underlining in kitty that puts the underline right up to the baseline." src="https://user-images.githubusercontent.com/45116528/159133952-7ad483c6-6006-4f0a-9941-a6c86da997e9.png">

`adjust_line_height` and `adjust_baseline` seem to have no effect. I searched through this repo's discussions and couldn't find any mention of this. This is on macOS 12.

**Comment by page-down** [comment, marked as answer]
Maybe you are interested in the following PR.

https://github.com/kovidgoyal/kitty/pull/4723

Unfortunately it is not ready yet, so it has not been merged.

**Reply by geball** [reply]
Much appreciated, @page-down. I searched through the issues but not the PRs. Thank you.
